tuf/client_rework/updater_rework.py

```diff
@@ -542,14 +357,13 @@ def _preorder_depth_first_walk(self, target_filepath) -> Dict:
 
             # The metadata for 'role_name' must be downloaded/updated before
             # its targets, delegations, and child roles can be inspected.
-            # self._metadata['current'][role_name] is currently missing.
             # _refresh_targets_metadata() does not refresh 'targets.json', it
```

Comment:
The rest of the comment is also out-of-date as it references functions not existing in this client version.